### PR TITLE
allowed users error

### PR DESCRIPTION
**Pull request by ThatGalNatalie**
PR Checklist:

- [x] add a short description of what's changed to the top of the `CHANGELOG.md`
- [ ] add/update tests (or don't, for reasons explained below)

## Describe this PR
Set the initial value of this.users and this.invitations was undefined on mount 
Made this.allowedUsers a computed prop to use the nullish coalescing operator /check for total allowed users

### CODE REVIEW

**Comment by znicholasbrown** [comment]
This removed the warning but I think in the wrong way - testing on a tenant with 10 allowed users (from the license terms) but more than 10 users already in the tenant and no warning is displayed

**Review by znicholasbrown — DISMISSED** [review]
This removed the warning but I think in the wrong way - testing on a tenant with 10 allowed users (from the license terms) but more than 10 users already in the tenant and no warning is displayed

File: src/pages/TeamSettings/Members.vue [hunk 1]
```diff
@@ -81,8 +81,11 @@ export default {
     ...mapGetters('tenant', ['tenant']),
     ...mapGetters('user', ['user']),
     ...mapGetters('license', ['license', 'hasPermission', 'allowedUsers']),
+    totalAllowedUsers() {
+      return this.allowedUsers() ?? Infinity
+    },
     insufficientUsers() {
-      return this.users >= this.allowedUsers() ?? Infinity
+      return this.users >= this.totalAllowedUsers
```
Comment (ThatGalNatalie) [inline src/pages/TeamSettings/Members.vue:88]: Looks like `this.users` is undefined. Will fix now

**Review by znicholasbrown — APPROVED** [review]
Works great, thanks for the changes @ThatGalNatalie 👍🏻

**Pull request merged**